### PR TITLE
FEAT: Add ban command

### PR DESCRIPTION
**Pull request by bunnechi**
User gets banned through the database

### CODE REVIEW

File: src/commands/command.py [hunk 1]
```diff
@@ -516,3 +516,25 @@ async def handle_logger_off(client, txt_channel, author, msg, message) \
...
+        role=[config.ADMIN_ROLE]
+    )
+    async def handle_ban(client, txt_channel, author, msg, message) \
+            -> "Ban a player":
+        args = msg.split(" ")
+        if len(args) < 3:
+            await txt_channel.send("Usage: !ban <character> <reason>")
+            return False
+        player = args[1]
+        i = 2
```
Comment (KOOKIIEStudios) [inline src/commands/command.py]: Consider using List slices instead of `while`

File: src/commands/command.py [hunk 2]
```diff
@@ -516,3 +516,25 @@ async def handle_logger_off(client, txt_channel, author, msg, message) \
...
+        reason = ""
+        while i < len(args):
```
Comment (KOOKIIEStudios) [inline src/commands/command.py]: Use descriptive variable names, as per Google conventions - avoid `i`

File: src/commands/command.py [hunk 3]
```diff
@@ -516,3 +516,25 @@ async def handle_logger_off(client, txt_channel, author, msg, message) \
...
             return False
+
+    @staticmethod
+    @command(
+        cmd=["ban"],
+        role=[config.ADMIN_ROLE]
+    )
+    async def handle_ban(client, txt_channel, author, msg, message) \
+            -> "Ban a player":
+        args = msg.split(" ")
```
Comment (Bratah123) [inline src/commands/command.py]: msg.split() does the same as msg.split(" ")!

File: src/settings/config.py [hunk 4]
```diff
@@ -82,6 +82,7 @@
     "credits",
     "credit",
     "info",
+    "ban"
```
Comment (KOOKIIEStudios) [inline src/settings/config.py]: Append trailing comma, as per Google convention

File: src/settings/config.py [hunk 5]
```diff
@@ -101,6 +102,7 @@
     'handle_setgmlevel': True,
     'handle_give_vp': True,
     'handle_giveaway': False,
+    'handle_ban': False
```
Comment (KOOKIIEStudios) [inline src/settings/config.py]: Append trailing comma, as per Google convention

**Review by Bratah123 — APPROVED** [review]
LGTM!

**Review by KOOKIIEStudios — APPROVED** [review]
LGTM - raised style issues resolved.  
This is usually where I say :shipit: , but she never tested it before PR-ing so...

**Pull request merged**